docs(k8s): add mention that pool scaling not available during upgrade (#4194)
scaleway/docs-content: containers/kubernetes/how-to/upgrade-kubernetes-version.mdx

    <Message type="note">
      It is not possible to downgrade your Kubernetes version once it has been upgraded.
    </Message>
    <Message type="note">

>        Resizing cluster pools is not allowed during the upgrade, provision accordingly.

Reviewer: ```suggestion
       Cluster pools cannot be resized during the upgrade process. Provision resources accordingly.
```
